modules/op-supported-parameters-tekton-chains-configuration.adoc

```diff
@@ -271,15 +281,23 @@ If you configure the `kms` signature backend, set the KMS configuration, includi
 |
 
 | `signers.kms.auth.token`
-| Authentication token for the KMS server (the value of `VAULT_TOKEN`).
+| Authentication token for the KMS server (the value of `VAULT_TOKEN`). Providing the token as plain-text configuration might be insecure. Use the alternative `signers.kms.auth.token-dir` configuration setting for production environments.
 |
 |
 
+
+| `signers.kms.auth.token-dir`
+| The directory where a file named `KMS_AUTH_TOKEN` is located. This file contains the authentication token for the KMS server (the value of `VAULT_TOKEN`).
+Provide this file as a secret and configure mounting this file for the {tekton-chains} controller, as described in _Creating and mounting the KMS authentication token secret_.
+| Example value: `/etc/kms-secrets`
```

Comment:
Can you please change this based on the updated docs here https://github.com/tektoncd/chains/blob/main/docs/config.md#kms-oidc-and-spire-configuration